packages/phpstan-rules/packages/cognitive-complexity/tests/Rules/ClassLikeCognitiveComplexityRule/ClassLikeCognitiveComplexityRuleTest.php

```diff
@@ -26,11 +26,13 @@ public function test(string $filePath, array $expectedErrorMessagesWithLines): v
     public function provideDataForTest(): Iterator
     {
         $errorMessage = sprintf(ClassLikeCognitiveComplexityRule::ERROR_MESSAGE, 'Class', 54, 50);
-
         yield [__DIR__ . '/Fixture/ClassWithManyComplexMethods.php', [[$errorMessage, 7]]];
 
-        $errorMessage = sprintf(ClassLikeCognitiveComplexityRule::ERROR_MESSAGE, 'Class', 9, 5);
+        $errorMessage = sprintf(ClassLikeCognitiveComplexityRule::ERROR_MESSAGE, 'Class', 34, 5);
         yield [__DIR__ . '/Fixture/SimpleCommand.php', [[$errorMessage, 9]]];
+
+        $errorMessage = sprintf(ClassLikeCognitiveComplexityRule::ERROR_MESSAGE, 'Class', 44, 5);
+        yield [__DIR__ . '/Fixture/NonFinalClass.php', [[$errorMessage, 9]]];
```

Comment:
Could you separate this to a new test case file, so we have the original behavior in own file?